addons/pos_coupon/static/src/js/coupon.js

```diff
@@ -275,6 +275,12 @@ odoo.define('pos_coupon.pos', function (require) {
             }
             return [...nonRewardLines, ...rewardLines];
         }
+
```

Comment:
Thanks for the change, can you remove the spacing here and below to respect the writing style of the file?